cmd/spiffe-helper/config/config.go

```diff
@@ -46,28 +48,36 @@ type JWTConfig struct {
 	JWTSVIDFilename string `hcl:"jwt_svid_file_name"`
 }
 
+func ParseFlags() (string, bool) {
+	configFile := flag.String("config", "helper.conf", "<configFile> Configuration file path")
+	daemonModeFlag := flag.Bool(daemonModeFlagName, true, "Exit once the requested objects are retrieved")
```

Comment:
Usually the doc for a flag describes what it does if true I think. So this is worded backwards from what one might expect.

Reply:
updated description.